docs/CodeTransformation.md

```diff
@@ -22,109 +22,114 @@ If you override the `transform` configuration option `babel-jest` will no longer
 You can write your own transformer. The API of a transformer is as follows:
 
 ```ts
-interface SyncTransformer<OptionType = unknown> {
-  /**
-   * Indicates if the transformer is capable of instrumenting the code for code coverage.
-   *
-   * If V8 coverage is _not_ active, and this is `true`, Jest will assume the code is instrumented.
-   * If V8 coverage is _not_ active, and this is `false`. Jest will instrument the code returned by this transformer using Babel.
-   */
+// This version of the interface you are seeing on the website has been trimmed down for brevity
+// For the full definition, see `packages/jest-transform/src/types.ts` in https://github.com/facebook/jest 
+// (taking care in choosing the right tag/commit for your version of Jest)
+
+export interface CallerTransformOptions {
```

Comment:
can drop `export` on all of these, methinks

Reply:
Still, just a docs-only change right? For reasons mentioned below :)

Reply:
yes! 😀